action/concrete_factory_test.go

```diff
@@ -137,10 +137,15 @@ var _ = Describe("concreteFactory", func() {
 			Expect(action).To(Equal(NewCreateDisk(diskCreator)))
 		})
 
-		XIt("delete_disk", func() {
+		It("delete_disk", func() {
```

Comment:
We should probably start adding more descriptive `It(...)` names and descriptions. Seems to me that "delete_disk" is very generic and may something like `It("delete the attached disk", ...)` would be more appropriate?

If you agree, let's start doing this in subsequent PRs.